sample_files/pre-commit-13.0/.pre-commit-config.yaml

```
@@ -0,0 +1,62 @@
+exclude: "^setup/"
+default_language_version:
+  python: python3
+repos:
+- repo: https://github.com/ambv/black
+  rev: 19.3b0
+  hooks:
+  - id: black
+- repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v2.3.0
+  hooks:
+  - id: trailing-whitespace
+    # exclude autogenerated files
+    exclude: /README\.rst$|\.pot?$
+  - id: end-of-file-fixer
+    # exclude autogenerated files
+    exclude: /README\.rst$|\.pot?$
+  - id: debug-statements
+  - id: flake8
+    name: flake8 except __init__.py
+    exclude: /__init__\.py$
+    additional_dependencies: ["flake8-bugbear==19.8.0"]
+  - id: flake8
+    name: flake8 only __init__.py
+    args: ["--extend-ignore=F401"]  # ignore unused imports in __init__.py
+    files: /__init__\.py$
+    additional_dependencies: ["flake8-bugbear==19.8.0"]
+- repo: https://github.com/pre-commit/mirrors-pylint
+  rev: v2.3.1
+  hooks:
+  - id: pylint
+    name: pylint with optional checks
+    args: ["--rcfile=.pylintrc", "--exit-zero"]
+    verbose: true
+    additional_dependencies: ["pylint-odoo==3.0.3"]
+- repo: https://github.com/pre-commit/mirrors-pylint
+  rev: v2.3.1
+  hooks:
+  - id: pylint
+    name: pylint with mandatory checks
+    args: ["--rcfile=.pylintrc-mandatory"]
+    additional_dependencies: ["pylint-odoo==3.0.3"]
+- repo: https://github.com/asottile/pyupgrade
+  rev: v1.24.0
+  hooks:
+  - id: pyupgrade
+- repo: https://github.com/asottile/seed-isort-config
+  rev: v1.9.3
+  hooks:
+  - id: seed-isort-config
+- repo: https://github.com/pre-commit/mirrors-isort
+  rev: v4.3.21
+  hooks:
+  - id: isort
+    name: isort except __init__.py
+    exclude: /__init__\.py$
+- repo: https://github.com/pre-commit/mirrors-eslint
+  rev: v6.5.1
+  hooks:
+  - id: eslint
+    verbose: true
+    exclude: /static/src/lib/
```

Is this a linter or a fixer? ESLint supports both where possible, and only linter where not. That should be enabled IMHO.

Also, shouldn't this line be like this?:

```suggestion
    exclude: */static/src/lib/
```

---

It's actually a regex not a glob pattern, so it's correct.

---

Which makes we think we should exclude /static/src/lib globally, right?